doc/source/front-end/structure-du-site.rst

```diff
@@ -276,3 +276,17 @@ Le menu pour mobile est généré à partir de l'en-tête et de la barre latéra
 
 .. figure:: ../images/design/menu-mobile.png
    :align: center
+
+Design fluide (responsive design)
+=================================
+
+Le design est prévu dès le début pour être adapté à un maximum de supports (à priori tous). Ainsi, il faudra prendre garde à quelques détails si l'on souhaite un comportement idéal sur tablettes et mobiles.
+
+Une différence notable sur les petits appareils est que la barre de navigation latérale gauche est masquée par défaut. Elle est disponible au glissé (swipe) du bord gauche vers la droite, ou directement via l'icône en haut.
+
+.. note::
+    Sur les anciens navigateurs mobiles (navigateur natif Android), ce menu n'est pas mis en place au profit de liens directement visibles mais de fait plus encombrants. Le soucis vient de l'impossibilité de gérer correctement le menu latéral. Le menu latéral est alors placé tout en bas de la page sur ces navigateurs.
+
+Cette barre latérale sera à soigner, sa structure HTML est relativement complexe principalement au niveau des attributs et classes à placer pour obtenir le comportement souhaité.
+
+Le site est codé selon la pratique mobile first, c'est à dire que le design est prévu pour les mobiles en premier : dans le code le style par défaut est celui destiné aux mobiles. Des feuilles de styles avec media-queries viendront par la suite surcharger les propriétés pour les appareils plus grands. Cette pratique permet un gain de performances.
```

Comment:
> Cette pratique permet un gain de performances.

Pour autant que je sache, pas vraiment. C’est surtout que c’est
généralement plus simple : https://zellwk.com/blog/how-to-write-mobile-first-css/

En gros, le site va avoir deux styles distincts : Celui pour
mobile (`m`), et celui pour bureau (`b`). `m` partage du style
avec `b`. On peut définir un style `g` qui s’applique à la fois
sur mobile et sur bureau, et qui est commun aux styles `m` et
`b`. On peut aussi dire que `m` et `b` héritent de `g`. Le fait
est que généralement, par rapport à `g`, le style `m` est une
mince surcouche contrairement à `b`. Du coup, dans le CSS, il est
**généralement** plus simple de d’abord définir le style mobile
et d’écraser ce qu’il ne va pas pour avoir le style bureau que de
faire l’inverse.

(En espérant que mon explication appliquant la théorie des
ensembles aux feuilles de styles CSS n’est pas trop ridicule)

Reply:
Je viens de l'enlever, mais ce n'était pas de moi, juste mergé ce qui était sur le gh-pages.

Reply:
cf https://zestedesavoir.github.io/zds-site/#/general/esprit-guidelines :)